org.eclipse.ui.console/src/org/eclipse/ui/console/TextConsoleViewer.java

```diff
@@ -93,6 +93,8 @@ public class TextConsoleViewer extends SourceViewer implements LineStyleListener
 
 	private boolean consoleAutoScrollLock = true;
 
+	private int mouseOffset;
```

Comment:
Initial value should be -1, otherwise if mouse was never moved, changed code will still try to find and create a link at first line.